### PR TITLE
DOC-4561 fixes for RDI Helm docs

### PR DESCRIPTION
**Pull request by andy-stark-redis**
[DOC-4561](https://redislabs.atlassian.net/browse/DOC-4561)

Let me know if this needs more emphasis or explanation, and if you have a better value for `JWT_SECRET_KEY` than `yourKey`, I'll add it in.

[DOC-4561]: https://redislabs.atlassian.net/browse/DOC-4561?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dwdougherty — APPROVED** [review]
LGTM.

File: content/integrate/redis-data-integration/installation/install-k8s.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ To pull images from a local registry, you must provide the image pull secret and
 1.  Decompress the tar file:
 
     ```bash
-	tar -xvf rdi-k8s-<rdi-tag>.tar.gz 
+	tar -xvf rdi-<rdi-tag>.tar.gz
```
Comment (dwdougherty) [inline content/integrate/redis-data-integration/installation/install-k8s.md:97]: It looks like a tab character was used before "tar". I suggest replacing this with spaces. Here's an octal dump of the above snippet:

```
0000000    1   .           D   e   c   o   m   p   r   e   s   s       t
0000020    h   e       t   a   r       f   i   l   e   :  \n  \n        
0000040            `   `   `   b   a   s   h  \n  \t   t   a   r       -
0000060    x   v   f       r   d   i   -   <   r   d   i   -   t   a   g
0000100    >   .   t   a   r   .   g   z      \n                   `   `
0000120    `                                                            
0000121
```

In VS Code, the tar line does not line up with the indent of the line above.

**Review by tahan42 — COMMENTED** [review]
LGTM!

**Comment by loriotpiroloriol** [comment]
In `values.yaml` there are several instances where values apply to `mTLS`, but comments state `TLS`, specifically `RDI_REDIS_CERT` and `RDI_REDIS_KEY` in section `rdiSysSecret` and `cert` and `key` in section `rdiDbSSLSecret`.

**Comment by loriotpiroloriol** [comment]
We should make it clear that `JWT_SECRET_KEY` should have 32 ASCII characters (256 bits, best practice).

**Comment by loriotpiroloriol** [comment]
There is no mention about how to get `redis-di` installed to work alongside the Helm installation.

**Comment by loriotpiroloriol** [comment]
We should make it clear which values need to be set in `values.yaml` as a minimum, i.e. `RDI_REDIS_HOST`, `RDI_REDIS_PORT` and `RDI_REDIS_PASSWORD`.

**Review by yaronp68 — APPROVED** [review]

**Pull request merged**